config/deploy.rb

```diff
@@ -28,6 +28,35 @@
 
 # Default value for linked_dirs is []
 # append :linked_dirs, "log", "tmp/pids", "tmp/cache", "tmp/sockets", "public/system"
+desc "Marks the server(s) to be removed from the loadbalancer"
+  task :remove_from_nginx do
+    count = 0
+    on roles(:app) do
+      count += 1
+    end
+    if count > 1
```

Comment:
@kayiwa Can you pull this stuff from https://github.com/pulibrary/figgy/pull/6539/files#diff-3d066bc83eaf220ed7d8276202a78415dfe66ddf285f3382273b86d5845ab9c4R92 instead, so we can turn off more than one at a time?